opennft/config.py

```diff
@@ -7,17 +7,17 @@
 
 """
 
-import os
+from pathlib import Path
 import pyqtgraph as pg
 
 
 LOG_LEVEL = 'DEBUG'
 
-ROOT_PATH = os.path.abspath(os.path.dirname(__file__))
-UI_PATH = os.path.join(ROOT_PATH, 'ui')
-PLUGIN_PATH = os.path.join(ROOT_PATH, 'plugins')
-OpenNFT_ICON = os.path.join(ROOT_PATH, 'ui', 'images', 'appicon.png')
-MATLAB_FUNCTIONS_PATH = os.path.join(ROOT_PATH, 'matlab')
+ROOT_PATH = Path(__file__).absolute().resolve().parent
+UI_PATH = ROOT_PATH.joinpath('ui')
```

Comment:
use `/` instead of joinpatch:
```python
UI_PATH = ROOT_PATH / 'ui'
```

Reply:
I think `joinpath `is more portable. Supporting Windows may not be on the table but we should not exclude it.

Reply:
@tiborauer `/` operator works fine on any supported platforms.

```python
Path('aaa') / 'bbb' / 'ccc'
WindowsPath('aaa/bbb/ccc')
```

Reply:
Thanks, @espdev. I did not know that.

Reply:
Replaced with `/`